### DISCUSSION TITLE
Customer access to only see reporting of their project

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4284] [Questions & Answers]
I like to grant some customers access with a special row. 

They should be able to:  
1. see their Projects 
2. see tracked times for this project

They shouldn't see:
1. internal ratings or ratings at all (only hours)
2. any other project the developers participated in

**BTW:** 
Is there a way to get reportings via API? This would be very handy to create automated reports (e.g. via cron, etc.) to send customers an overview of the current worked hours in the Project.

**Comment by kevinpapst** [comment]
https://www.kimai.org/store/vettersolutions-shared-project-timesheets-bundle.html

BTW: No, not possible yet.
